backend/src/main/java/com/github/makitango/curiocity/event/exceptions/EventNotFoundException.java

```diff
@@ -0,0 +1,14 @@
+package com.github.makitango.curiocity.event.exceptions;
+
+public class EventNotFoundException extends RuntimeException {
+
+    public static final String DEFAULT_MESSAGE = "Event not found";
+
+    public EventNotFoundException(String message) {
+        super(message);
+    }
+
+    public EventNotFoundException() {
+        super(DEFAULT_MESSAGE);
+    }
```

Comment:
Ich würde die Nachricht direkt einfügen
```suggestion
public class EventNotFoundException extends RuntimeException {
    public EventNotFoundException(String message) {
        super(message);
    }

    public EventNotFoundException() {
        super("Event not found");
    }
```